src/styles/blocks/button.scss

```diff
@@ -0,0 +1,33 @@
+.button {
+  display: flex;
+  width: 100%;
+  height: 56px;
+  color: #fff;
+  box-sizing: border-box;
+  cursor: pointer;
+  margin-bottom: 20px;
+
+  text-decoration: none;
+  align-items: center;
+  justify-content: center;
+  background: #292929;
+  border-radius: 40px;
+  transition: all 0.2s;
+
+  &:hover {
+    font-weight: 700;
+    color: #292929;
+    background-color: transparent;
+    border: 3px solid #292929;
```

Comment:
Check your hover transition. It's not smooth because you add border for hover effect, but for static one you do not have border.
So there is a way how to fix it: use `box-shadow` instead of border
`box-shadow: inset 0px 0px 0px 3px #292929;`